commands/minigame.js

```diff
@@ -0,0 +1,98 @@
+// Spec for minigame command (based off earlier discussion):
+// 2 new classes; Minigame and Question (maybe more if you inheret from Question or use question-type)
+// Questions are stored in a questions.json
+
+import { readFileSync } from "fs";
+import Discord from "discord.js";
+
+const State = Object.freeze({
+  "off": 0,
+  "starting": 1,
+  "inprogress": 2
```

Comment:
No need for quotes, I would also make these things uppercase
```suggestion
  OFF: 0,
  STARTING: 1,
  IN_PROGRESS: 2
```